runtime/doc/lua.txt

```diff
@@ -876,6 +876,34 @@ vim.region({bufnr}, {pos1}, {pos2}, {type}, {inclusive})       *vim.region()*
         whether the selection is inclusive or not, into a zero-indexed table 
         of linewise selections of the form `{linenr = {startcol, endcol}}` .
 
+                                             *vim.register_keystroke_callback()*
+vim.register_keystroke_callback({fn}, {ns_id})
+        Register a lua {fn} with an {ns_id} to be run after every keystroke.
```

Comment:
`-w` was mentioned in the PR discussion, that's a hint that it should be mentioned in the docs here (and the commit message).